Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md

```diff
@@ -6495,21 +6524,69 @@ Retrieve vulnerability details for a specific ID and host. Supported with the Cr
 | --- | --- | --- | --- |  --- | --- |  --- | --- |  --- | --- |
 | CVE-20212-2222 |  host | 1 | Server | ip |  |  | site | 5.5 |  |
 
+
+### cs-falcon-get-events
+
+***
+Get events from CrowdStrike Falcon. This command is intended for development and debugging purposes and should be used with caution as it may create duplicate events. Note that pushing events is supported on Cortex XSIAM only.
```

Comment:
```suggestion
Get events from CrowdStrike Falcon. This command is intended for development and debugging purposes and should be used with caution as it may create duplicate events. Note that pushing events is supported in Cortex XSIAM only.
```